doc/classes/EditorImportPlugin.xml

```diff
@@ -241,7 +241,80 @@
 			<param index="2" name="custom_importer" type="String" default="&quot;&quot;" />
 			<param index="3" name="generator_parameters" type="Variant" default="null" />
 			<description>
-				This function can only be called during the [method _import] callback and it allows manually importing resources from it. This is useful when the imported file generates external resources that require importing (as example, images). Custom parameters for the ".import" file can be passed via the [param custom_options]. Additionally, in cases where multiple importers can handle a file, the [param custom_importer] can be specified to force a specific one. This function performs a resource import and returns immediately with a success or error code. [param generator_parameters] defines optional extra metadata which will be stored as [code skip-lint]generator_parameters[/code] in the [code]remap[/code] section of the [code].import[/code] file, for example to store a md5 hash of the source data.
+				This function can only be called during the [method _import] callback and it allows manually importing resources from it. This is useful when the imported file generates external resources that require importing (as example, images). Custom parameters for the ".import" file can be passed via the [param custom_options]. Additionally, in cases where multiple importers can handle a file, the [param custom_importer] can be specified to force a specific one. This function performs a resource import and returns immediately with a success or error code. [param generator_parameters] defines optional extra metadata which will be stored as [code skip-lint]generator_parameters[/code] in the [code]remap[/code] section of the [code].import[/code] file, for example to store a md5 hash of the source data. For example:
+				[codeblocks]
+				[gdscript]
+				var _editor: EditorInterface
+
+				func _import(source_file, save_path, options, r_platform_variants, r_gen_files):
+				    # Make textures folder if required.
+				    var source_file_folder: String = source_file.rsplit("/", true, 1)[0]
+				    var textures_folder: String = "%s/textures" % source_file_folder
+				    DirAccess.make_dir_recursive_absolute(textures_folder)
+
+				    # Simulate image export by copying the source file to the textures folder.
+				    var export_path: String = "%s/icon.png" % textures_folder
+				    var export_image_file: FileAccess = FileAccess.open(export_path, FileAccess.WRITE)
+				    export_image_file.store_buffer(FileAccess.get_file_as_bytes("res://game/my_icon.png"))
+				    export_image_file.close()
+
+				    # Refresh view of exported file.
+				    var editor_file_system: EditorFileSystem = _editor.get_resource_filesystem()
```

Comment:
As discussed in the original PR, the `_editor` variable is never assigned so this would fail. It should probably be changed the same way it was changed in the C# version:

```suggestion
				    var editor_file_system: EditorFileSystem = EditorInterface.get_resource_filesystem()
```

Unlike the C# version, the GDScript code doesn't need to use a `Singleton` property.